docs/CONTRIBUTING.md

```diff
@@ -16,30 +16,55 @@ please [see here](http://bit.ly/argo-wf-cmty-mtng).
 
 ## Contributor Meetings (twice monthly)
 
-A weekly opportunity for committers and maintainers of Workflows and Events to discuss their current work and
+An opportunity for committers and maintainers of Workflows and Events to discuss their current work and
 talk about what’s next. Feel free to join us! For Contributor Meeting information, minutes and recordings
 please [see here](https://bit.ly/argo-data-weekly).
 
+## Slack
+
+You can join the following channels on [CNCF Slack](https://argoproj.github.io/community/join-slack):
+* `#argo-workflows`: discussions focused mainly on use of Argo Workflows
+* `#argo-wf-contributors`: discussions focused mainly on development of Argo Workflows
+
 ## How To Contribute
 
 We're always looking for contributors.
 
-* Documentation - something missing or unclear? Please submit a pull request!
-* Code contribution - investigate
-  a [good first issue](https://github.com/argoproj/argo-workflows/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
-  , or anything not assigned.
+### Roles
```

Comment:
So the "Roles" section and the "Sustainability Effort" are actually not specific to "How To Contribute" (the "how" in particular), so I wouldn't put them as subsections of it. I'm fine with them either being above or below the entire "How To Contribute" section